File: reverse/solutions/reverse.cs
using System;
using System.Collections.Generic;
class MainClass {
  public static void Main (string[] args) {
    string the_string = "LaunchCode";
    string reversed_string = "";
    for (int i = the_string.Length - 1; i >= 0; i--) {
      reversed_string += the_string[i];
    }
    Console.WriteLine(reversed_string);
    
  }
}

File: reverse/solutions/reverse.java
import java.util.*;
class Main {
  public static void main(String[] args) {
    
    String the_string = "LaunchCode";
    char[] chars = the_string.toCharArray();
    String reversed_string = "";
    for (int i = chars.length - 1; i >= 0; i--) {
      reversed_string += chars[i];
    }
    
    System.out.println(reversed_string);
    
  }
}